### PR TITLE
fix: issues after latest merge

### PR DESCRIPTION
**Pull request by 0dm**

### CODE REVIEW

File: openadapt/models.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """This module defines the models used in the OpenAdapt system."""
 
-from typing import Union
+from typing import Any, Union
```
Comment (Mustaballer) [inline openadapt/models.py:3]: Can you remove Any here? It's not used

**Review by Mustaballer — CHANGES_REQUESTED** [review]
Had one comment, besides that looks good!

**Review by Mustaballer — APPROVED** [review]

File: openadapt/app/__init__.py [hunk 2]
```diff
@@ -1,13 +0,0 @@
...
-This package contains modules for the OpenAdapt application.
-
-Example usage:
-    from openadapt.app import run_app
-
-    run_app()
-"""
-from openadapt.app.main import run_app
-
-if __name__ == "__main__":
```
Comment (Mustaballer) [inline openadapt/app/__init__.py:12]: Is there a reason why you deleted this? It was like this before my PR was merged
```python
from openadapt.app.main import run_app

if __name__ == "__main__":
    run_app()
```

Comment (0dm) [reply openadapt/app/__init__.py:12]: it was causing some issues, and isn't really needed. It was removed a while ago I think

Comment (Mustaballer) [reply openadapt/app/__init__.py:12]: @0dm Hey Aaron, can you run ```black --check .```  to see if any files need reformatting and ```flake8 .```  to see if any lint errors come up. If those pass then the PR should be ready for merge.

Comment (0dm) [reply openadapt/app/__init__.py:12]: yep, there was a few things missing -- should be good now

**Comment by Mustaballer** [comment]
lgtm!

File: openadapt/models.py [hunk 3]
```diff
@@ -19,7 +19,9 @@ class ForceFloat(sa.TypeDecorator):
     impl = sa.Numeric(10, 2, asdecimal=False)
     cache_ok = True
 
-    def process_result_value(self, value: Union[int, float, str, None]) -> float | None:
+    def process_result_value(
+        self, value: Union[int, float, str, None], dialict: str
```
Comment (abrichr) [inline openadapt/models.py:23]: Can we please use `|` instead of `Union`?

File: openadapt/visualize.py [hunk 4]
```diff
@@ -317,7 +317,11 @@ def main() -> None:
     logger.info(f"{fname_out=}")
     output_file(fname_out, title=title)
 
-    # result = show(layout(rows,))
+    result = show(  # noqa: F841
```
Comment (abrichr) [inline openadapt/visualize.py:320]: Can you please clarify what the pragma statement is for?

Comment (Mustaballer) [reply openadapt/visualize.py:320]: @abrichr The error thrown from flake8 is F841
> [Local variable name is assigned to but never used (F841)](https://www.flake8rules.com/rules/F841.html)

**Pull request merged**